.github/sendgrid.js

```diff
@@ -0,0 +1,30 @@
+#! /usr/bin/env node
+
+const sgMail = require('@sendgrid/mail');
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+const fs = require('fs'),
+    filename = 'hyscale_version.txt',
+    fileType = 'application/txt',
+    data = fs.readFileSync(filename);
+
+const msg = {
+    to: 'engineering@hyscale.io',
```

Comment:
Add them both eng DL and Srujan id as properties. Let us hike them from users.

Reply:
comma seperated multiple users not working already tested